### ISSUE TITLE
ServiceLoadBalancer should support simple GET on Ingress ips is failing on kubernetes-e2e-gke-ci-reboot

### ISSUE THREAD

**Issue by roberthbailey** [kubernetes/kubernetes#16602]
This looks like it might be a configuration error on the jenkins machine:

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/serviceloadbalancers.go:239
Expected error:
    <*os.PathError | 0xc2088bc750>: {
        Op: "open",
        Path: "/jenkins-master-data/jobs/kubernetes-e2e-gke-ci-reboot/workspace/kubernetes/test/e2e/testing-manifests/haproxyrc.yaml",
        Err: 0x2,
    }
    open /jenkins-master-data/jobs/kubernetes-e2e-gke-ci-reboot/workspace/kubernetes/test/e2e/testing-manifests/haproxyrc.yaml: no such file or directory
not to have occurred
```

**Comment by ixdy** [comment]
This test is broken and has never worked on Jenkins. It's skipped for GCE but not GKE (https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e.sh#L108).

The issue is that the test depends on files in `test/e2e/testing-manifests`, but these are not included in the kubernetes-test tarball. Most likely `hack/lib/golang.sh` needs to be updated to include these files.

The test will work locally because you have access to the full kubernetes repository, but this is not true on Jenkins. Running `hack/e2e-from-release.sh` will simulate the Jenkins environment locally.

@kubernetes/goog-testing

**Comment by bprashanth** [comment]
sorry I thought I'd disabled it: https://github.com/kubernetes/kubernetes/pull/16625

This test pulls an image from /contrib and is meant so people from the community can run ad-hoc tests against a kube cluster with the haproxy loadbalancer. Running it on our internal jenkins builder was never the goal (we eventually will, but we're not there yet with the Ingress api). However since all the kube-up scripts are in main repo, and it uses shared library functions with other loadbalancer tests that do run regularly, it made more sense putting it here.

**Comment by bprashanth** [comment]
But it is also really confusing that it worked locally and not in production. I'll fix one way or another anyway when I have some time (either modify golang.sh or embed the manifests in the test).

**Comment by ixdy** [comment]
Yeah, you're not the first to run into issues with the tarballs not including all test dependencies.

**Comment by ihmccreery** [comment]
Ping @bprashanth.

**Comment by ihmccreery** [comment]
(This is very much related to #10548.)
